### PR TITLE
Sample database main

### PR DESCRIPTION
**Pull request by kriks7iitk**
1. Added sample database functionality in CE
2. Changed delete icon for sample datasource list
3. Changed empty state for editor container
4. Added sample application template using sample data

### CODE REVIEW

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: server/src/services/sample_db.service.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import { ConfigService } from '@nestjs/config';
+import { Injectable } from '@nestjs/common';
+import { DataSource } from '../../src/entities/data_source.entity';
```
Comment (gsmithun4) [inline server/src/services/sample_db.service.ts:3]:
```suggestion
import { DataSource } from '../entities/data_source.entity';
```

File: server/package.json [hunk 2]
```diff
@@ -85,6 +86,7 @@
     "tsconfig-paths": "^3.10.1",
     "typeorm": "^0.2.38",
     "ws": "^7.5.5",
+    "xlsx": "^0.18.5",
```
Comment (gsmithun4) [inline server/package.json:89]: revert this file changes

File: server/scripts/populate-sample-db.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  return new Client({
+    user: envVars.SAMPLE_PG_DB_USER,
+    host: envVars.SAMPLE_PG_DB_HOST,
+    database: envVars.SAMPLE_DB,
+    password: envVars.SAMPLE_PG_DB_PASS,
+    port: envVars.SAMPLE_PG_DB_PORT,
+  });
+}
+
+const folderPath = path.join(__dirname, '../src/assets/sample-data-excel');
```
Comment (gsmithun4) [inline server/scripts/populate-sample-db.ts:17]: should get the data from json file

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/Editor/editor.theme.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.empty-box-cont{
```
Comment (gsmithun4) [inline frontend/src/Editor/editor.theme.scss:1]: if it is specific to sample data source , add it to sample data source related css file

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/Editor/DataSourceManager/dataSourceManager.theme.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
+@import '../../_styles/colors.scss';
+
```
Comment (gsmithun4) [inline frontend/src/Editor/DataSourceManager/dataSourceManager.theme.scss:2]: follow lint rules

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/Editor/editor.theme.scss (hunk 4 above)
Comment (kriks7iitk) [reply frontend/src/Editor/editor.theme.scss:1]: No this is specific to editor only

File: frontend/src/Editor/DataSourceManager/dataSourceManager.theme.scss (hunk 5 above)
Comment (kriks7iitk) [reply frontend/src/Editor/DataSourceManager/dataSourceManager.theme.scss:2]: So SCSS done take short path notation from config file

**Review by gsmithun4 — DISMISSED** [review]

**Review by gsmithun4 — APPROVED** [review]

**Pull request merged**